### ISSUE TITLE
remove "rename variable" and "delete variable" from the change by and set to variable and list blocks

### ISSUE THREAD

**Issue by ntlrsk** [LLK/scratch-gui#3218]
In 3.0 the menu options to rename and delete a variable are now in the variable block menus (*change by*and *set to*) in addition to in the reporter block. Although this makes them more visible, it also makes the basic functionality of the *change by* and *set to* more confusing. It isn't clear which variable is to be renamed or deleted. The option to rename and delete a variable is a more advanced action, so I think is better to just stay in the reporter block.  

For example, in resources where suggesting add a score variable, the drop-down menu for selecting "score" is now confusing.

![image](https://user-images.githubusercontent.com/4721032/45968357-b68b9700-bffe-11e8-8b44-8e8bbe4875e4.png)

![image](https://user-images.githubusercontent.com/4721032/45968319-9cea4f80-bffe-11e8-8fa9-ef55da6c3855.png)

**Comment by towerofnix** [comment]
>Additional note: The list block wording talks about renaming the "variable" - should that be "list"?

This is fixed in PR scratch-blocks#1715 (https://github.com/LLK/scratch-blocks/pull/1715/commits/ac0dabe3ed85ef60681d4c3faec426c99f4c8011).

**Comment by ntlrsk** [comment]
To clarify - in the reporter block menu, instead of listing the different variable names at the end, the options to Rename "[variable name]" variable and "Delete the "[variable name]" variable would go there. 

Current Scratch 3.0 with variable names listed at the end:
![image](https://user-images.githubusercontent.com/4721032/45973046-5b13d600-c00b-11e8-852a-62f707797f8e.png)

**Comment by alexmotoc** [comment]
I also noticed there are some inconsistencies - the different variable names appear at the end of the context menu whereas for lists the behaviour is not the same (the only options are Duplicate, Add Comment and Delete Block).

**Comment by MasterOfTheTiger** [comment]
I believe the 2.0 behavior is better. This is rather confusing.

**Comment by benjiwheeler** [comment]
I can see the argument for including rename and delete in this list, as a place where some users will look for them. Lots of younger and less computer savvy users don't know about context menus.

That said, I think 1) delete should maybe be removed, and 2) we should at least but a separator in the menu between the variable names and the "rename" choice.

**Comment by ntlrsk** [comment]
@benjiwheeler I agree at minimum providing a dividing line.  A remaining problem then is that it's hard to know which one you're wanting to delete or rename. In Scratch 1.4 there was not an option to rename a variable - it's more of a power user function, I think.  For younger and less computer savvy I think the more intuitive thing to do is to make a new variable and drag that in.

**Comment by ntlrsk** [comment]
The variable block menus in Scratch 3.0 are still more complicated to understand and use than in Scratch 2.0.
Recommendation to match variable menus in 2.0. 
Specifically for reporter blocks use: "rename variable / delete variable" 
For other variable blocks- only list the names of the variables.
